clang/test/SemaSYCL/accessor-type-diagnostics.cpp

```diff
@@ -1,9 +1,9 @@
 // RUN: %clang_cc1 -triple spir64 -fsycl-is-device -verify \
 // RUN:  -aux-triple x86_64-unknown-linux-gnu -fsyntax-only       \
-// RUN:  -Wno-sycl-2017-compat  %s
+// RUN:   %s
```

Comment:
I would suggest to squash this into the previous line. The command was split to make it fit into 80-symbol limit, but now that we have an argument removed, we can drop the 3rd line. The same applies to lines 5 and 6